renovate.json5

```diff
@@ -5,7 +5,7 @@
     ],
     baseBranches: [
         "master",
-        //    "renovate-playground"
+        "renovate-playground"
```

Comment:
what is this for?

Reply:
see [here](https://github.com/hivemq/hivemq-amazon-kinesis-extension/pull/70#discussion_r1281786476).